docs: split Tracing Policy concept page into subpages

mtardy opened the pull request:
Now the Tracing Policy documentation splits into three pages:
- an example of policy, shortly explained
- a hook point page about how to use kprobes, tracepoints, etc.
- a selector page about filters and actions

See [preview](https://deploy-preview-1377--tetragon.netlify.app/docs/concepts/tracing-policy/).

mtardy commented:
Need to add a link to the examples directory in the repository to the example page

kkourt commented on docs/content/en/docs/concepts/tracing-policy/example.md:
````diff
@@ -0,0 +1,144 @@
+      type: "file"
+```
+
+The beginning of the specification describe the hook point to use, here we
+are using a kprobe, hooking on the kernel function `fd_install`. We indicate
+that it's not a syscall (which is by the way unecessary since false is the
+default). Then we specify the argument of the specified function symbol to be
+able to extract them, and optionally perform filtering.
````
```suggestion
The beginning of the specification describes the hook point to use. In this case, it is 
a kprobe, hooking on the kernel function `fd_install`. We indicate
that it's not a syscall (can be omitted, since this is the default). We also specify arguments and their type, which will  instruct Tetragon to extract them and include them in any resulting events, as well enable filtering based on their values in subsequent selectors.
```

kkourt replied:
Also just noticed that syscall is true by default: https://github.com/cilium/tetragon/blob/main/pkg/k8s/apis/cilium.io/v1alpha1/types.go#L118-L120

mtardy replied:
I removed that sentence.

kkourt reviewed (commented):
Thanks!

Generally looks good to me, but I have some comments / suggestions. Please see below.

mtardy commented on docs/content/en/docs/concepts/tracing-policy/example.md:
```diff
@@ -0,0 +1,144 @@
+        operator: "Equal"
+        values:
+        - "/etc/passwd"
+        - "/etc/shadow"
```
maybe it's better if we don't use enforcement on the example? I don't know on one side I wanted something simple but on the other side I wanted to showoff Tetragon's capabilities. Maybe removing sigkill is better.

kkourt replied:
I think using sigkill is good for illustration purposes: i) it illustrates how one can use actions, and ii) you can just see the application being killed rather than an event which is more immediate.

mtardy replied:
Yep I used a dummy `/tmp/tetragon` file instead

mtardy commented on docs/content/en/docs/concepts/tracing-policy/example.md:
````diff
@@ -0,0 +1,144 @@
+      matchAction:
+      - action: Sigkill
+```
+
````
I added a warning.

mtardy commented:
> Thanks!
> 
> Generally looks good to me, but I have some comments / suggestions. Please see below.

I integrated your suggestions, changed the example, and added warnings.

kkourt reviewed (approved):
🚀
